scripts/config/pages.js

```diff
@@ -21,6 +21,7 @@ module.exports = [
 
     ['trading',                                 'app/trade/trading',                           'default',    'SmartTrader'],
 
+    ['new_account/digital_options',             'app/new_account/digital_options',             'default',    'Digital Options'],
```

Comment:
(question) this `new_account` page is is reserved for Real Account creation, not sure `digital_options` belong here. Would you please elaborate?